src/Feature/Multisite/code/Views/Multisite/SwitchSite.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using Sitecore.Feature.Multisite
+@using Sitecore.Mvc
+
+@model Sitecore.Feature.Multisite.Models.SiteDefinitions
+
+<div class="dropdown">
+    <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        @if (Model.Current == null)
+        {
+            <span>@Request.Url.Host</span>
+        }
+        else
+        {
+            <span>@Model.Current.Name</span>
+        }
+        <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+        @foreach (var site in Model.Sites)

```

Comment:
View receive filtered list of site definition. SiteDefinition list is filtered by the SiteDefinitionsRepository by checking ShowInMenu field of the _SiteConfiguration template.